app/build.gradle

```diff
@@ -25,24 +32,29 @@ android {
         versionCode 3
         versionName "1.1.1"
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
-        manifestPlaceholders = [
-                // The hostname is used when building the intent-filter, so the TWA is able to
-                // handle Intents to open https://svgomg.firebaseapp.com.
-                hostName: "svgomg.firebaseapp.com",
-                defaultUrl: "https://svgomg.firebaseapp.com",
-                launcherName: "SVGOMG TWA",
-                // This variable below expresses the relationship between the app and the site,
-                // as documented in the TWA documentation at
-                // https://developers.google.com/web/updates/2017/10/using-twa#set_up_digital_asset_links_in_an_android_app
-                // and is injected into the AndroidManifest.xml
-                assetStatements: '[{ "relation": ["delegate_permission/common.handle_all_urls"], ' +
-                        '"target": {"namespace": "web", "site": "https://svgomg.firebaseapp.com"}}]'
-        ]
+
+        // The name for the application on the Android Launcher
+        resValue "string", "appName", twaManifest.name
+
+        // The URL that will be used when launching the TWA from the Android Launcher
+        resValue "string", "launchUrl", "https://" + twaManifest.hostName + twaManifest.launchUrl
+
+        // The hostname is used when building the intent-filter, so the TWA is able to
+        // handle Intents to open https://svgomg.firebaseapp.com.
+        resValue "string", "hostName", twaManifest.hostName
+
+        // This variable below expresses the relationship between the app and the site,
+        // as documented in the TWA documentation at
+        // https://developers.google.com/web/updates/2017/10/using-twa#set_up_digital_asset_links_in_an_android_app
+        // and is injected into the AndroidManifest.xml
+        resValue "string", "assetStatements",
+                '[{ "relation": ["delegate_permission/common.handle_all_urls"],' +
+                        '"target": {"namespace": "web", "site": "https://'+twaManifest.hostName+'"}}]'
```

Comment:
nit: can you put some spaces around the '+' operators?

Reply:
done